src/sql/parts/notebook/constants.ts

```diff
@@ -0,0 +1,6 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export const doubleClickToEdit = 'Double-click to edit';
```

Comment:
This needs to be a localized string. Please put back into the textCell.component.ts class and use `localize('doubleClickEdit', 'Double-click to edit')`. Thanks!